### PR TITLE
Add Git:Fetch (prune)

### PR DESCRIPTION
**Pull request by rscott**
My attempt at fixing #59693

### CODE REVIEW

File: extensions/git/package.json [hunk 1]
```diff
@@ -265,6 +265,11 @@
...
         "category": "Git"
       },
+      {
+        "command": "git.fetchPrune",
+        "title": "%command.fetchPrune%",
+        "category": "Git"
+      },
       {
         "command": "git.fetchAll",
         "title": "%command.fetchAll%",
```
Comment (oneslash) [inline extensions/git/package.json]: also don't forget to add condition in `menus.menus.commandPalette`

File: extensions/git/src/git.ts [hunk 2]
```diff
@@ -1172,6 +1172,11 @@ export class Repository {
 			args.push('--all');
 		}
 
+		if (options.prune) {
+			args.push('-p');
```
Comment (oneslash) [inline extensions/git/src/git.ts]: I believe more understandable would be to use full argument name `--prune`

File: extensions/git/package.json (hunk 1 above)
Comment (rscott) [reply extensions/git/package.json]: Thanks @oneslash, I definitely missed this

File: extensions/git/src/git.ts (hunk 2 above)
Comment (rscott) [reply extensions/git/src/git.ts]: Agree, just habit of typing the short form in the CLI 😄

**Comment by joaomoreno** [comment]
Thanks! :beers:

**Pull request merged**